src/CashlinkApp.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+    <div id="app">
+        <header class="logo">
+            <span class="nq-icon nimiq-logo"></span>
+            <span class="logo-wordmark">Nimiq</span>
+            <span class="logo-subtitle">Cashlink</span>
+            <div class="flex-grow"></div>
+            <a class="nq-button-s" href="https://nimiq.com/#splash" target="_blank">What is Nimiq?</a>
+        </header>
+        <div v-if="loading" class="loading">
+            <LoadingSpinner/>
+        </div>
+        <CashlinkReceive v-else/>
+    </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import { LoadingSpinner } from '@nimiq/vue-components';
+import CashlinkReceive from './views/CashlinkReceive.vue';
+import { loadNimiq } from './lib/Helpers';
+
+import '@nimiq/style/nimiq-style.min.css';
+import '@nimiq/vue-components/dist/NimiqVueComponents.css';
+
+@Component({components: {LoadingSpinner, CashlinkReceive}})
+export default class CashlinkApp extends Vue {
+    private loading = true;
+
+    public async created() {
+        await this.$store.dispatch('initWallets');
+        await loadNimiq();
```

Comment:
This is literally the top component that gets rendered on page load.

Reply:
Is the whole cashlink app in a single bundle, such that we wouldn't benefit from doing this already in `cashlink.ts`?
We could call `Nimiq.WasmHelper.doImport()` in index.html.
This would then already start loading the wasm such that later when `loadNimiq` is called part of the work is already done.

Reply:
This `CashlinkApp` is the root component of a separate Cashlink app bundle, yes. That's why it's loaded immediately and runs immediately.